code/modules/projectiles/special.dm

```diff
@@ -191,6 +191,15 @@
 		FC.dissipation_ticker = rand(0,dissipation_variance)
 		FC.launch()
 
+/datum/projectile/special/spreader/buckshot_burst/nails
```

Comment:
Consider adding proper projectile stats (power, damage type) for pointblanking

Reply:
Should be fixed by 757ac52

Reply:
Intentional that the pointblank is outright stronger than regular firing at 'point blank range'? (note that general behavior for pointblanking is to be the same as a direct hit)

Reply:
Pointblanking doesn't give any bleed, so I buffed on point-blank range to compensate for that :v